share/palettes.txt

```diff
@@ -0,0 +1,7224 @@
+palette batlow RGBf
```

Comment:
Consider adding comments to indicate the source of the palettes (https://zenodo.org/records/8409685?) and/or changing the name of the file, e.g. to `palettes-crameri.txt`.

We should probably also mention the license at the end of our `NOTICES` file?

In the future, we could potentially have different `palettes-*.txt` files in `share/` from different sources.

Reply:
added in [190e5c4](https://github.com/GLVis/glvis/pull/289/commits/190e5c40e4d5212e6819da5214ef4164a2cc8c4d)